tests/ha/pacemaker_cts_regression.pm

```diff
@@ -0,0 +1,37 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2016-2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Execute regression tests with pacemaker-cts
+# Maintainer: Julien Adamek <jadamek@suse.com>
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use utils 'zypper_call';
+use testapi;
+use lockapi;
```

Comment:
`testapi` and `lockapi` are not needed here, please remove and add it in the other PR if needed.